frontend/src/metabase/query_builder/components/QueryModals/QueryModals.tsx

```diff
@@ -214,7 +217,8 @@ export function QueryModals({
           originalQuestion={originalQuestion}
           onSave={handleSaveAndClose}
           onCreate={handleCreateAndClose}
-          onCancel={onCloseModal}
+          onClose={onCloseModal}
+          opened={true}
```

Comment:
Just curious, will we ever have any uses of this component where the `opened` prop would ever be `false`?

Reply:
exactly. I hope we end up in a world where we use the modal controls as they should be used rather than this current state, but for now here we are 😅